### ISSUE TITLE
Fix false positive on MiKo_3092

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#388]
Event registration or deregistration should not trigger a violation, as in following snippet.
```c#
lock (this)
{
    item.PropertyChanged -= OnPropertyChanged;
}
```
